#Fix error: error: cannot convert 'std::nullptr_t' to 'Py_ssize_t' {aka 'long int'} in initialization
diff --git a/apps/ivr-python2/IvrAudio.cpp b/apps/ivr-python2/IvrAudio.cpp
index 8522f25d..77cecc7c 100644
--- a/apps/ivr-python2/IvrAudio.cpp
+++ b/apps/ivr-python2/IvrAudio.cpp
@@ -315,7 +315,7 @@ static PyGetSetDef IvrAudioFile_getseters[] = {
     
 PyTypeObject IvrAudioFileType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrAudioFile",        /*tp_name*/
   sizeof(IvrAudioFile),      /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrAudioMixIn.cpp b/apps/ivr-python2/IvrAudioMixIn.cpp
index f4971fbc..241a0546 100644
--- a/apps/ivr-python2/IvrAudioMixIn.cpp
+++ b/apps/ivr-python2/IvrAudioMixIn.cpp
@@ -95,7 +95,7 @@ static PyGetSetDef IvrAudioMixIn_getseters[] = {
     
 PyTypeObject IvrAudioMixInType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrAudioMixIn",        /*tp_name*/
   sizeof(IvrAudioMixIn),      /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrDialogBase.cpp b/apps/ivr-python2/IvrDialogBase.cpp
index 3ca5defd..c3e76768 100644
--- a/apps/ivr-python2/IvrDialogBase.cpp
+++ b/apps/ivr-python2/IvrDialogBase.cpp
@@ -744,7 +744,7 @@ static PyGetSetDef IvrDialogBase_getset[] = {
 
 PyTypeObject IvrDialogBaseType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrDialogBase",       /*tp_name*/
   sizeof(IvrDialogBase),     /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrNullAudio.cpp b/apps/ivr-python2/IvrNullAudio.cpp
index d1e6d254..562f0da2 100644
--- a/apps/ivr-python2/IvrNullAudio.cpp
+++ b/apps/ivr-python2/IvrNullAudio.cpp
@@ -90,7 +90,7 @@ static PyGetSetDef IvrNullAudio_getseters[] = {
     
 PyTypeObject IvrNullAudioType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrNullAudio",        /*tp_name*/
   sizeof(IvrNullAudio),      /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrSipDialog.cpp b/apps/ivr-python2/IvrSipDialog.cpp
index 1b788bc9..75b9b802 100644
--- a/apps/ivr-python2/IvrSipDialog.cpp
+++ b/apps/ivr-python2/IvrSipDialog.cpp
@@ -121,7 +121,7 @@ static PyGetSetDef IvrSipDialog_getset[] = {
 
 PyTypeObject IvrSipDialogType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrSipDialog",        /*tp_name*/
   sizeof(IvrSipDialog),      /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrSipReply.cpp b/apps/ivr-python2/IvrSipReply.cpp
index b66942c3..26a5c2de 100644
--- a/apps/ivr-python2/IvrSipReply.cpp
+++ b/apps/ivr-python2/IvrSipReply.cpp
@@ -109,7 +109,7 @@ static PyGetSetDef IvrSipReply_getset[] = {
 
 PyTypeObject IvrSipReplyType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrSipReply",        /*tp_name*/
   sizeof(IvrSipReply),      /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrSipRequest.cpp b/apps/ivr-python2/IvrSipRequest.cpp
index f07cfe8f..788be248 100644
--- a/apps/ivr-python2/IvrSipRequest.cpp
+++ b/apps/ivr-python2/IvrSipRequest.cpp
@@ -183,7 +183,7 @@ static PyGetSetDef IvrSipRequest_getset[] = {
 
 PyTypeObject IvrSipRequestType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrSipRequest",        /*tp_name*/
   sizeof(IvrSipRequest),      /*tp_basicsize*/
diff --git a/apps/ivr-python2/IvrUAC.cpp b/apps/ivr-python2/IvrUAC.cpp
index 2ea3b076..c90b7e36 100644
--- a/apps/ivr-python2/IvrUAC.cpp
+++ b/apps/ivr-python2/IvrUAC.cpp
@@ -92,7 +92,7 @@ static PyMethodDef IvrUAC_methods[] = {
 };
 
 PyTypeObject IvrUACType = {	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.IvrUAC",              /*tp_name*/
   sizeof(IvrUAC),            /*tp_basicsize*/
diff --git a/apps/ivr/IvrAudio.cpp b/apps/ivr/IvrAudio.cpp
index c123b522..59d411de 100644
--- a/apps/ivr/IvrAudio.cpp
+++ b/apps/ivr/IvrAudio.cpp
@@ -249,7 +249,7 @@ static PyGetSetDef IvrAudioFile_getseters[] = {
     
 PyTypeObject IvrAudioFileType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrAudioFile",        /*tp_name*/
   sizeof(IvrAudioFile),      /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrAudioMixIn.cpp b/apps/ivr/IvrAudioMixIn.cpp
index 6894a150..a446416b 100644
--- a/apps/ivr/IvrAudioMixIn.cpp
+++ b/apps/ivr/IvrAudioMixIn.cpp
@@ -95,7 +95,7 @@ static PyGetSetDef IvrAudioMixIn_getseters[] = {
     
 PyTypeObject IvrAudioMixInType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrAudioMixIn",        /*tp_name*/
   sizeof(IvrAudioMixIn),      /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrDialogBase.cpp b/apps/ivr/IvrDialogBase.cpp
index f59cb604..b8dbd52d 100644
--- a/apps/ivr/IvrDialogBase.cpp
+++ b/apps/ivr/IvrDialogBase.cpp
@@ -744,7 +744,7 @@ static PyGetSetDef IvrDialogBase_getset[] = {
 
 PyTypeObject IvrDialogBaseType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrDialogBase",       /*tp_name*/
   sizeof(IvrDialogBase),     /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrNullAudio.cpp b/apps/ivr/IvrNullAudio.cpp
index 1410aee8..1f3e8c8d 100644
--- a/apps/ivr/IvrNullAudio.cpp
+++ b/apps/ivr/IvrNullAudio.cpp
@@ -90,7 +90,7 @@ static PyGetSetDef IvrNullAudio_getseters[] = {
     
 PyTypeObject IvrNullAudioType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrNullAudio",        /*tp_name*/
   sizeof(IvrNullAudio),      /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrSipDialog.cpp b/apps/ivr/IvrSipDialog.cpp
index 127650a6..714e8498 100644
--- a/apps/ivr/IvrSipDialog.cpp
+++ b/apps/ivr/IvrSipDialog.cpp
@@ -121,7 +121,7 @@ static PyGetSetDef IvrSipDialog_getset[] = {
 
 PyTypeObject IvrSipDialogType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrSipDialog",        /*tp_name*/
   sizeof(IvrSipDialog),      /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrSipReply.cpp b/apps/ivr/IvrSipReply.cpp
index ccdecfee..73c7266a 100644
--- a/apps/ivr/IvrSipReply.cpp
+++ b/apps/ivr/IvrSipReply.cpp
@@ -109,7 +109,7 @@ static PyGetSetDef IvrSipReply_getset[] = {
 
 PyTypeObject IvrSipReplyType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrSipReply",        /*tp_name*/
   sizeof(IvrSipReply),      /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrSipRequest.cpp b/apps/ivr/IvrSipRequest.cpp
index 77aa2db6..605cd2bd 100644
--- a/apps/ivr/IvrSipRequest.cpp
+++ b/apps/ivr/IvrSipRequest.cpp
@@ -183,7 +183,7 @@ static PyGetSetDef IvrSipRequest_getset[] = {
 
 PyTypeObject IvrSipRequestType = {
     
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrSipRequest",        /*tp_name*/
   sizeof(IvrSipRequest),      /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/ivr/IvrUAC.cpp b/apps/ivr/IvrUAC.cpp
index 76af3294..e57dad3d 100644
--- a/apps/ivr/IvrUAC.cpp
+++ b/apps/ivr/IvrUAC.cpp
@@ -92,7 +92,7 @@ static PyMethodDef IvrUAC_methods[] = {
 };
 
 PyTypeObject IvrUACType = {	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   "ivr.IvrUAC",              /*tp_name*/
   sizeof(IvrUAC),            /*tp_basicsize*/
   0,                         /*tp_itemsize*/
diff --git a/apps/py_sems/PySemsAudio.cpp b/apps/py_sems/PySemsAudio.cpp
index bfcbefcf..6048849d 100644
--- a/apps/py_sems/PySemsAudio.cpp
+++ b/apps/py_sems/PySemsAudio.cpp
@@ -282,7 +282,7 @@ static PyGetSetDef PySemsAudioFile_getseters[] = {
     
 PyTypeObject PySemsAudioFileType = {
 	
-  PyObject_HEAD_INIT(NULL)
+  PyVarObject_HEAD_INIT(NULL, 0)
   0,                         /*ob_size*/
   "ivr.PySemsAudioFile",        /*tp_name*/
   sizeof(PySemsAudioFile),      /*tp_basicsize*/
